projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -117,7 +144,13 @@ const LineChart: FC< LineChartProps > = ( {
 	} );
 
 	return (
-		<div className={ clsx( 'line-chart', styles[ 'line-chart' ], className ) }>
+		<div
+			className={ clsx( styles[ 'line-chart' ], className ) }
+			data-testid="line-chart"
+			style={ { width, height } }
```

Comment:
Could you please explain more why the width and height are needed here?

Reply:
Ah, good catch. The style={{ width, height }} inline style was added to control the container’s dimensions, but it’s unnecessary because the visx XYChart component already manages sizing with its width and height props.